packages/scss/src/components/_table.scss

```diff
@@ -149,19 +149,52 @@
 		.table-body-row-cell:first-child {
 			padding-left: 2rem;
 			position: relative;
+		}
+		
+		// .table-body-row-cell:first-child {
+		// 	padding-left: 2rem;
+		// 	position: relative;
+
+		// 	&::before {
+		// 		@include makeIcon("drag");
+		// 		bottom: 0;
+		// 		color: _color("text.light");
+		// 		cursor: move;
+		// 		font-size: 1.3rem;
+		// 		height: 2rem;
+		// 		left: 0;
+		// 		margin: auto;
+		// 		position: absolute;
+		// 		top: 0;
+		// 	}
+		// }
+	}
 
-			&::before {
-				@include makeIcon("drag");
-				bottom: 0;
-				color: _color("text.light");
-				cursor: move;
-				font-size: 1.3rem;
-				height: 2rem;
-				left: 0;
-				margin: auto;
-				position: absolute;
-				top: 0;
-			}
+	.table-body-row-cell-handler {
+		position: absolute;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		width: 2rem;
+		cursor: move;
+	  
+		&:hover {
+		  &::after {
+			color: _color('text.light');
+		  }
+		}
+	  
+		&::after {
```

Comment:
You have an absolute positionned element in a absolute positionned element. It is not optimal. Isn't it possible to have the after not being positioned absolute ?